stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -81,6 +82,107 @@ public static boolean isBlank(String value) {
     }
 
     /**
+     * Returns a {@link CardBrand} corresponding to a partial card number, or {@link Card#UNKNOWN}
+     * if the card brand can't be determined from the input value.
```

Comment:
Double-checking that we've chosen not to return a list of valid values. Returning a single value makes sense right now, but returning a list has it's own benefits:

I could envision building a slick UI on top of  `getPossibleCardTypes`:

* Empty string: All Card brands are highlighted
* 3 is typed: `[Diners, Amex, JCB]` are highlighted.
* then 4 is typed, and `Amex` is highlighted